src/pages/LandingLogin/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+import api from '../../services/api';
+import '../LandingLogin/styles.css';
+
+import userIcon from '../../assets/images/userIcon.svg';
+import Union from '../../assets/images/Union.svg';
+import lock from '../../assets/images/lock.svg';
+
+import Logo from '../../components/Logo';
+
+export default function LandingLogin() {
+    const [email, setEmail] = useState("");
+    const [senha, setSenha] = useState("");
+
+    async function handleLogin() {
+
+        try {
+            const response = await api.get('/users', { email, senha });
+
+            if(!response) {
+                throw() => {
+                    alert("usuario não encontrado na base de dados");
+                }
+            }
```

Comment:
Eu não sei se esse alert fica muito bom para o UX, inda mais quando for criar as telas responsivas. Uma opção seria usar algo parecido com esses: https://react-bootstrap.github.io/components/alerts/.

Mas não precisa ser agora, a gente pode deixar pra refatorar proxima sprint